docs/components/navigator.md

```diff
@@ -0,0 +1,115 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# gamepad.navigator
+
+This component listens to the `gamepadconnected` and `gamepaddisconnected` events in the browser. When at least one
+gamepad is connected, it reads inputs from them at a particular frequency and stores those input values in the
+component's model data. The component stops reading these inputs when the last gamepad is disconnected.
+
+## Using this grade
+
+This grade can work without passing any option to it. An instance of the `navigator` does not require any options to be
```

Comment:
The first and second sentences are somewhat redundant.  I would change both to something like "You can change the configuration options by either extending the component or by passing your own options.  If no options are passed, the defaults are used."